src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -378,12 +382,12 @@ public void update(double time, double dt) {
                     signal.isFieldOriented());
         } else {
             synchronized (stateLock) {
-
                 if (getAntiTip() && driveSignal != null) {
                     signal = new HolonomicDriveSignal( // create updated drive signal
-                            driveSignal.getTranslation().rotateBy(driveSignal.isFieldOriented() ? // flatten
-                                    getAngle() : Rotation2.ZERO) // same code as other block
-                                    .add(tipController.update(getGyroscopeXY())), // anti tip stuff
+                            accelLimiter.calculate( // vector accel limiter
+                                    driveSignal.getTranslation().rotateBy(driveSignal.isFieldOriented() ? // flatten
+                                            getAngle() : Rotation2.ZERO) // same code as other block
+                                            .add(tipController.update(getGyroscopeXY()))), // anti tip stuff
```

Comment:
Will the active anti-tip portion be more effective outside the slewlimiter?  The slew limiter seems like it will help prevent tips due to fast acceleration or stops.  Will slew run the risk of damping and delaying the anti-tip signal to the wheels once the robot is past the tip threshold and it is trying to recover?

Reply:
good point ill fix